




 ### Suman start run indiv. @Mon Jun 06 2016 17:58:07 GMT-0700 (PDT) ### 

 ### Filename: /Users/amills/WebstormProjects/oresoftware/suman/test/build-tests/test6.test.js
 ### Beginning of run at 1465261087050 = [Mon Jun 06 2016 17:58:07 GMT-0700 (PDT)]
 ### Command = ["/usr/local/Cellar/node/6.2.0/bin/node","/Users/amills/WebstormProjects/oresoftware/suman/lib/poolio-worker.js","/Users/amills/WebstormProjects/oresoftware/suman/test/build-tests/test6.test.js"]


	 => Suman non-fatal error => Error in hook and "fatal" option for the hook is set to false => 
	Error: fail this test thing
	    at TestSuiteBase.<anonymous> (/Users/amills/WebstormProjects/oresoftware/suman/test/build-tests/test6.test.js:22:15)
	    at /Users/amills/WebstormProjects/oresoftware/suman/lib/test-suite-helpers/make-handle-each.js:154:52
	    at _combinedTickCallback (internal/process/next_tick.js:67:7)
	    at process._tickDomainCallback (internal/process/next_tick.js:122:9)	 => Suman non-fatal error => Error in hook and "fatal" option for the hook is set to false => 
	Error: fail this test thing
	    at TestSuiteBase.<anonymous> (/Users/amills/WebstormProjects/oresoftware/suman/test/build-tests/test6.test.js:22:15)
	    at /Users/amills/WebstormProjects/oresoftware/suman/lib/test-suite-helpers/make-handle-each.js:154:52
	    at _combinedTickCallback (internal/process/next_tick.js:67:7)
	    at process._tickDomainCallback (internal/process/next_tick.js:122:9)	 => Suman non-fatal error => Error in hook and "fatal" option for the hook is set to false => 
	Error: fail this test thing
	    at TestSuiteBase.<anonymous> (/Users/amills/WebstormProjects/oresoftware/suman/test/build-tests/test6.test.js:22:15)
	    at /Users/amills/WebstormProjects/oresoftware/suman/lib/test-suite-helpers/make-handle-each.js:154:52
	    at _combinedTickCallback (internal/process/next_tick.js:67:7)
	    at process._tickDomainCallback (internal/process/next_tick.js:122:9)	 => Suman non-fatal error => Error in hook and "fatal" option for the hook is set to false => 
	Error: fail this test thing
	    at TestSuiteBase.<anonymous> (/Users/amills/WebstormProjects/oresoftware/suman/test/build-tests/test6.test.js:22:15)
	    at /Users/amills/WebstormProjects/oresoftware/suman/lib/test-suite-helpers/make-handle-each.js:154:52
	    at _combinedTickCallback (internal/process/next_tick.js:67:7)
	    at process._tickDomainCallback (internal/process/next_tick.js:122:9)	 => Suman non-fatal error => Error in hook and "fatal" option for the hook is set to false => 
	Error: fail this test thing
	    at TestSuiteBase.<anonymous> (/Users/amills/WebstormProjects/oresoftware/suman/test/build-tests/test6.test.js:22:15)
	    at /Users/amills/WebstormProjects/oresoftware/suman/lib/test-suite-helpers/make-handle-each.js:154:52
	    at _combinedTickCallback (internal/process/next_tick.js:67:7)
	    at process._tickDomainCallback (internal/process/next_tick.js:122:9)	 => Suman non-fatal error => Error in hook and "fatal" option for the hook is set to false => 
	Error: fail this test thing
	    at TestSuiteBase.<anonymous> (/Users/amills/WebstormProjects/oresoftware/suman/test/build-tests/test6.test.js:22:15)
	    at /Users/amills/WebstormProjects/oresoftware/suman/lib/test-suite-helpers/make-handle-each.js:154:52
	    at _combinedTickCallback (internal/process/next_tick.js:67:7)
	    at process._tickDomainCallback (internal/process/next_tick.js:122:9)	 => Suman non-fatal error => Error in hook and "fatal" option for the hook is set to false => 
	Error: fail this test thing
	    at TestSuiteBase.<anonymous> (/Users/amills/WebstormProjects/oresoftware/suman/test/build-tests/test6.test.js:22:15)
	    at /Users/amills/WebstormProjects/oresoftware/suman/lib/test-suite-helpers/make-handle-each.js:154:52
	    at _combinedTickCallback (internal/process/next_tick.js:67:7)
	    at process._tickDomainCallback (internal/process/next_tick.js:122:9)

 ### Suman end run ### 






